examples/semantic_segmentation/semantic_segmentation/seed_test_suite.py

```diff
@@ -51,10 +77,12 @@ def seed_complete_test_case(args: Namespace) -> TestCase:
 
 def main(args: Namespace) -> None:
     kolena.initialize(os.environ["KOLENA_TOKEN"], verbose=True)
+    test_suite_name = f"{DATASET} [person]"
```

Comment:
Should we update the test suite name to include stratification information?

Reply:
Any ideas on a name here?

Reply:
What do you think of f"{DATASET} [person] by image size"?